src/components/RatingsReviews/RatingBreakdown.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+
+function RatingBreakdown ({ meta }) {
+
+  function avgRating() {
+    var sum = 0;
+    var totalReviews = 0;
+    for (var starRating in meta.ratings) {
+      sum += starRating * parseInt(meta.ratings[starRating])
+      totalReviews += parseInt(meta.ratings[starRating])
+    }
+    return Math.round((sum / totalReviews) * 10) / 10
+  }
+
+  function recPercentage() {
+    var totalReviews = 0;
+    var yesCount = 0
+    for (var vote in meta.recommended) {
+      if (vote === 'true') {
+        yesCount += parseInt(meta.recommended[vote])
+        totalReviews += parseInt(meta.recommended[vote])
+      } else {
+        totalReviews += parseInt(meta.recommended[vote])
+      }
+    }
+    return Math.round((yesCount / totalReviews) * 100)
+  }
+
+
+  var totalRatings = 0;
+  var starPct = {};
+  var ratingsPerStar = {
+    5: 0,
+    4: 0,
+    3: 0,
+    2: 0,
+    1: 0
+  }
+
+  function starPercentage() {
+    for (var starRating in meta.ratings) {
+      ratingsPerStar[starRating] = parseInt(meta.ratings[starRating])
+      totalRatings += parseInt(meta.ratings[starRating])
+    }
+    for (var key in ratingsPerStar) {
+      starPct[key] = Math.round(ratingsPerStar[key] / totalRatings * 100)
+    }
+  }
+  starPercentage()
```

Comment:
When are you calling this function? Should it be on a specific update/ when the data changes? I'm worried this is going to be calling all the time if it's left open in the body of the component like this...